src/Evolution/Systems/GeneralizedHarmonic/Tags.hpp

```diff
@@ -149,34 +147,34 @@ struct EvolvedFieldsFromCharacteristicFields : db::SimpleTag {
  * `ThreeIndexConstraintCompute`, `FourIndexConstraintCompute`, and
  * `ConstraintEnergyCompute` respectively
  */
-template <size_t SpatialDim, typename Frame>
```

Comment:
don't change SpatialDim to Dim (multiple times in this file)

Reply:
Changed back, but note that the use of `Dim` and `SpatialDim` is very inconsistent, even in this file